README.md

```diff
@@ -16,3 +16,34 @@ Command                                         | Description
 ```$ ./scripts/feeds/install device-manager```  | Install only the package device-manager
 
 
+### Building CreatorKit packages using pre-compiled OpenWrt SDK for Ci40 (Marduk):
+
+1. First download the SDK from https://downloads.creatordev.io/pistachio/marduk/OpenWrt-SDK-0.9.4-pistachio-marduk_gcc-5.2.0_musl-1.1.11.Linux-x86_64.tar.bz2
+2. Extract the SDK and go to the extracted folder.
+
+          tar -xjvf OpenWrt-SDK-0.9.4-pistachio-marduk_gcc-5.2.0_musl-1.1.11.Linux-x86_64.tar.bz2
+          cd OpenWrt-SDK-0.9.4-pistachio-marduk_gcc-5.2.0_musl-1.1.11.Linux-x86_64.tar.bz2
+3. Check whether the feeds.conf.default file has an entry of https://github.com/CreatorKit/openwrt-ckt-feeds.git
```

Comment:
While verifying, I see that there is actually "src-link ckt ../../openwrt-ckt-feeds", so it is not the github URL https://github.com/CreatorKit/openwrt-ckt-feeds.git.  Is that still correct?